apps/web/src/pages/templates/workflow/digest/BackOffTooltipIcon.tsx

```diff
@@ -1,4 +1,89 @@
+import { useMantineTheme } from '@mantine/core';
+
 export const BackOffTooltipIcon = () => {
+  const { colorScheme } = useMantineTheme();
+
+  if (colorScheme === 'light') {
+    return (
+      <svg xmlns="http://www.w3.org/2000/svg" width="280" height="57" viewBox="0 0 280 57" fill="none">
+        <g clip-path="url(#clip0_809_136)">
+          <path
+            d="M15.566 20.2137C15.8816 20.4324 16.0472 20.8106 15.9878 21.1887L13.9878 34.1887C13.941 34.4918 13.7566 34.7574 13.4878 34.9074C13.2191 35.0574 12.8972 35.0762 12.6128 34.9574L8.87533 33.4043L6.7347 35.7199C6.45658 36.0231 6.01908 36.1231 5.6347 35.9731C5.25033 35.8231 5.00033 35.4512 5.00033 35.0387V32.4262C5.00033 32.3012 5.0472 32.1824 5.13158 32.0918L10.3691 26.3762C10.5503 26.1793 10.5441 25.8762 10.3566 25.6887C10.1691 25.5012 9.86595 25.4887 9.66908 25.6668L3.31283 31.3137L0.553451 29.9324C0.222201 29.7668 0.00970049 29.4356 0.000325486 29.0668C-0.00904951 28.6981 0.184701 28.3543 0.503451 28.1699L14.5035 20.1699C14.8378 19.9793 15.2503 19.9981 15.566 20.2137Z"
+            fill="#BEBECC"
+          />
+        </g>
+        <rect
+          x="20.25"
+          y="27.79"
+          width="51.5"
+          height="0.5"
+          rx="0.25"
+          stroke="#BEBECC"
+          stroke-width="0.5"
+          stroke-dasharray="2 2"
+        />
+        <g clip-path="url(#clip1_809_136)">
```

Comment:
```suggestion
          strokeWidth="0.5"
          strokeDasharray="2 2"
        />
        <g clipPath="url(#clip1_809_136)">
```

And those other places. If I'm not wrong, it causes some errors